doc/source/cluster/kubernetes/user-guides/kuberay-gcs-ft.md

```diff
@@ -141,7 +141,7 @@ KEYS *
 # 1) "864b004c-6305-42e3-ac46-adfa8eb6f752"
 
 # Step 6.5: Check the value of the key.
-HGETALL 864b004c-6305-42e3-ac46-adfa8eb6f752
+SCAN 0 MATCH 864b004c-6305-42e3-ac46-adfa8eb6f752*
```

Comment:
can you mention which release this command is changed from here?

Reply:
wdym "which release"?

Reply:
like it is changed from ray 2.35 or sth like that.

Reply:
done